lib/Service/DocumentService.php

```diff
@@ -152,17 +152,45 @@ public function createDocument(File $file): Document {
 	}
 
 	/**
-	 * @param $document
+	 * @param $documentId
 	 * @return ISimpleFile
 	 * @throws NotFoundException
 	 */
-	public function getStateFile($document): ISimpleFile {
+	public function getStateFile($documentId): ISimpleFile {
+		$filename = (string)$documentId . '.yjs';
 		if (!$this->ensureDocumentsFolder()) {
 			throw new NotFoundException('No app data folder present for text documents');
 		}
-		return $this->appData->getFolder('documents')->getFile((string) $document);
+		return $this->appData->getFolder('documents')->getFile($filename);
+	}
+
+	/**
+	 * @param $documentId
+	 * @return ISimpleFile
+	 * @throws NotPermittedException
+	 */
+	public function createStateFile($documentId): ISimpleFile {
```

Comment:
Would be nice to have the parameters typed.

```suggestion
	public function createStateFile(int $documentId): ISimpleFile {
```

Also `@param int $documentId` in the function annotations.